app/ports/tests/test_statistics.py

```diff
@@ -0,0 +1,61 @@
+import os
+import json
+import datetime
+
+from django.test import TestCase, Client
+from django.urls import reverse
+
+from ports.models import UUID, PortInstallation, Submission, Port
+from MacPorts.config import TEST_SUBMISSIONS, TEST_PORTINDEX_JSON
+
+
+class TestStatistics(TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    @classmethod
+    def setUpTestData(cls):
+        Port.load(TEST_PORTINDEX_JSON)
+
+    def test_submission(self):
+        submission_body = 'submission[data]={\
```

Comment:
I would probably use `"""` instead of having to escape everything.

Reply:
Thank you.